starter-code/src/components/CountryDetail.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext } from "react";
+import { CountryContext } from "../context/countries.context";
+import { Link } from "react-router-dom";
+
+export const CountryDetails = props => {
+  const { getCountry } = useContext(CountryContext);
+  const countryCode = props.match.params.cca3;
+  const country = getCountry(countryCode);
```

Comment:
Por un lado no entiendo muy bien porqué os habéis creado un context cuando simplemente habrías podido utilizar directamente
```
const getCountry = code => countries.filter(c => c.cca3 === code)[0];
```
en este componente sin tener que crear un context, por el otro lado interesante el intento, pero el con el `useContext` debería ser utilizado para englobar y servir esa variable de manera "Global", no simplemente para devolver un valor